src/vs/workbench/contrib/inlineChat/common/inlineChat.ts

```diff
@@ -148,6 +148,11 @@ export interface IInlineChatService {
 export const INLINE_CHAT_ID = 'interactiveEditor';
 export const INTERACTIVE_EDITOR_ACCESSIBILITY_HELP_ID = 'interactiveEditorAccessiblityHelp';
 
+export const enum EditMode {
```

Comment:
unrelated change?

Reply:
No, it is intentional. It fixes this typescript error: `Enum 'EditMode' used before its declaration`:

![enum-used-before-declaration](https://github.com/microsoft/vscode/assets/23744935/3c5214cd-68bc-49fb-ae1e-03f904fbfa33)